huaweicloud/resource_huaweicloud_lb_certificate_test.go

```diff
@@ -292,3 +315,74 @@ EOT
 }
 `, name)
 }
+
+func testAccLBV2CertificateConfig_withEpsId(name string) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_lb_certificate" "certificate_1" {
+  name                  = "%s"
+  description           = "terraform test certificate"
+  domain                = "www.elb.com"
+  enterprise_project_id = "%s"
+  private_key           = <<EOT
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Where did you get this certificate? Will there be a risk of leakage?

Reply:
This certificate has been in the test case for a long time. It seems the certificate is just for testing and there is no risk.